portugues/colabora/comdiferencas/diff/pull/desisto/desistir.md

```diff
@@ -0,0 +1 @@
+Quando você desiste você descobre que ainda precisa usar o programa que tem o bug e que não tem outra opção senão conseguir arrumar o bug.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.